packages/gatsby-theme/src/components/BlogHead/BlogHeadContainer.js

```diff
@@ -17,16 +27,71 @@ const BlogHeadContainer = ({ children, ...props }) => {
         flexDirection: "row",
         flexWrap: "wrap",
         justifyContent: "space-between",
+        ...sx,
       }}
       {...props}
     >
-      {children}
+      <Heading
+        as='h1'
+        sx={{ fontSize: ["4xl", "5xl"], mb: "small", minWidth: "100%", ...sx }}
+      >
+        {title}
+      </Heading>
+      <Box __css={{ pr: "medium" }}>
+        {to ? (
+          <Link
+            to={to}
+            ariaLabel={ariaLabel}
+            sx={{ color: "inherit", cursor: "pointer" }}
+          >
+            <Icon icon='user' sx={{ mr: "xsmall", pt: 2 }} />
```

Comment:
Can we create a `const` or a `variant` for this styles instead of repeating them please?

Reply:
@luisduenas Sure!! thank you!